### PR TITLE
UI: Fix 24h Toggle

### LINKED ISSUE #22: GUI Display issues
- [x]  The change view from 24 hour option doesnt toggle the view under firefox, chrome, etc, nor under qbundle. It also has a graphical black glitch in the drop down. https://github.com/PoC-Consortium/Qbundle/issues/67

- [x] Assets never load under Qbundle. It appears to be the wallet doesnt like the chromium pocket browser. https://github.com/PoC-Consortium/Qbundle/issues/66

### PR DESCRIPTION
**Pull request by harry1453**
Fixes #22

### CODE REVIEW

**Comment by harry1453** [comment]
It's horrible yeah, couldn't resist fixing a couple of things like the string cast

File: html/ui/js/brs.util.js [hunk 1]
```diff
@@ -618,7 +618,7 @@ var BRS = (function(BRS, $, undefined) {
                 }
                 res += " " + hours + ":" + minutes + ":" + seconds;
 
-                if (!BRS.settings["24_hour_format"]) {
+                if (BRS.settings["24_hour_format"] == 0) {
```
Comment (blankey1337) [inline html/ui/js/brs.util.js]: Typically you should never use double equals in JS. It was probably fine the way it was tbh.

File: html/ui/js/brs.util.js [hunk 2]
```diff
@@ -574,19 +574,19 @@ var BRS = (function(BRS, $, undefined) {
         }, no_escaping);
     };
     BRS.formatTimestamp = function(timestamp, date_only) {
+        var date;
         if (typeof timestamp === "object") {
             date = timestamp;
         } else {
             date = new Date(Date.UTC(2014, 7, 11, 2, 0, 0, 0) + timestamp * 1000);
         }
-        var date;
```
Comment (blankey1337) [inline html/ui/js/brs.util.js]: Pure functions ftw :D

File: html/ui/js/brs.util.js (hunk 1 above)
Comment (harry1453) [reply html/ui/js/brs.util.js]: This was actually the fix for it. An identical as opposed to an equals would also work. I suppose therefore it's not falsy but I'm not too acquainted with javascript.

**Pull request merged**